aws/data_source_aws_ec2_transit_gateway_vpc_attachment.go

```diff
@@ -16,6 +16,10 @@ func dataSourceAwsEc2TransitGatewayVpcAttachment() *schema.Resource {
 		Read: dataSourceAwsEc2TransitGatewayVpcAttachmentRead,
 
 		Schema: map[string]*schema.Schema{
+			"appliance_mode_support": {
```

Comment:
Needs documentation in `website/docs/d/ec2_transit_gateway_vpc_attachment.html.markdown` 👍

Reply:
Good spot, completely forgot that! Have updated the doc